src/rpc-method/types.ts

```diff
@@ -1483,7 +1483,8 @@ export const GetActionsRequest = {
     for (const rawActionInfo of rawActionInfoList) {
       const actionInfo = ({
         actHash: rawActionInfo.getActhash(),
-        blkHash: rawActionInfo.getBlkhash()
+        blkHash: rawActionInfo.getBlkhash(),
+        timestamp: rawActionInfo.getTimestamp(),
       } as any) as IActionInfo;
```

Comment:
please add type annotation to IActionInfo

Reply:
`timestamp` type definition added to `IActionInfo`.